src/app/right-hand-panel.js

```diff
@@ -36,20 +81,20 @@ export class RightHandPanel {
      * @memberof RightHandPanel
      */
     attached() {
+
         $(this.element).find("a").click((e) => {
             e.preventDefault();
-
             let hash = e.currentTarget.hash;
-
+            this.selected_tab = e.currentTarget.hash;
```

Comment:
I think we should combine the hash variable (above) and the selected_tab one. with the selected_tab, the local hash becomes kind of obsolete.